### ISSUE TITLE
Theme is broken

### ISSUE THREAD

**Issue by S0apyy** [maenDisease/Solana#8]
profile bar is no longer transparent:
![image](https://github.com/maenDisease/Solana/assets/122272512/f810a786-bfd8-46d2-85be-6ce059587ba8)

servers and folders in the dock are not centered anymore:
![image](https://github.com/maenDisease/Solana/assets/122272512/61c70177-189e-47d5-91ac-bee92a9a099e)

the normal chat window also doesnt have rounded edges anymore and the text bar where ur supposed to type has no more offset to the incoming chat: 
![image](https://github.com/maenDisease/Solana/assets/122272512/5f45165f-6e9c-4c93-9994-8591f16f58ac)

**Comment by Nevotic** [comment]
Experiencing the same issues. Happened after the latest Discord update.

**Comment by littlelapin** [comment]
echoing the above issues as well

**Comment by S0apyy** [comment]
update: title bar seems to not work now either:
![image](https://github.com/maenDisease/Solana/assets/122272512/8eb4120c-700d-4422-aff6-1d7d15a531df)

**Comment by Nevotic** [comment]
Workaround: Add Rounded Discord theme and turn it on together with Solana. Fixes the issues with the borders being square instead of round.

**Comment by S0apyy** [comment]
> Workaround: Add Rounded Discord theme and turn it on together with Solana. Fixes the issues with the borders being square instead of round.

ive also gotten someone to help me in fixing these issues and i can drop a css file soon to get everything back to working as intended

**Comment by S0apyy** [comment]
[Solana.txt](https://github.com/maenDisease/Solana/files/13064080/Solana.txt)

this should fix all issues

**Comment by littlelapin** [comment]
> [Solana.txt](https://github.com/maenDisease/Solana/files/13064080/Solana.txt)
> 
> this should fix all issues

edit: nvm i figured it out! ah tysm for the fix again!

**Comment by S0apyy** [comment]
> > [Solana.txt](https://github.com/maenDisease/Solana/files/13064080/Solana.txt)
> > 
> > this should fix all issues
> 
> edit: nvm i figured it out! ah tysm for the fix again!

its the whole theme so just overwrite it yeah

**Comment by Nevotic** [comment]
Absolute legend @S0apyy - It worked perfectly these past 3 days. Until Discord has now updated something again which breaks the theme.

**Comment by S0apyy** [comment]
> Absolute legend @S0apyy - It worked perfectly these past 3 days. Until Discord has now updated something again which breaks the theme.

ill look at it today but im not promising i can fix it

**Comment by Nevotic** [comment]
> > Absolute legend @S0apyy - It worked perfectly these past 3 days. Until Discord has now updated something again which breaks the theme.
> 
> ill look at it today but im not promising i can fix it

No rush! Whenever you got the time! Thank you

**Comment by S0apyy** [comment]
> > > Absolute legend @S0apyy - It worked perfectly these past 3 days. Until Discord has now updated something again which breaks the theme.
> > 
> > ill look at it today but im not promising i can fix it
> 
> No rush! Whenever you got the time! Thank you

discord pushed out an update that breaks everything so possibility of me fixing this is like 1%

**Comment by Nevotic** [comment]
> > > > Absolute legend @S0apyy - It worked perfectly these past 3 days. Until Discord has now updated something again which breaks the theme.
> > > 
> > > 
> > > ill look at it today but im not promising i can fix it
> > 
> > 
> > No rush! Whenever you got the time! Thank you
> 
> discord pushed out an update that breaks everything so possibility of me fixing this is like 1%

Yeah the update Discord pushed basically broke everything. From themes to plugins. But no worries, thanks for the fix you made before. Let's hope the theme maker can come up with an update.

**Comment by S0apyy** [comment]
Fixed version: 
[solana.theme.txt](https://github.com/maenDisease/Solana/files/13169077/solana.theme.txt)

@Nevotic

**Comment by Nevotic** [comment]
> Fixed version: [solana.theme.txt](https://github.com/maenDisease/Solana/files/13169077/solana.theme.txt)
> 
> @Nevotic

You are an absolute legend!! Thank you so much

**Comment by S0apyy** [comment]
Missed some things for if you wanted bubble chat or not

fix for bubble chat(put it underneath every other line of code): 
[bubble.txt](https://github.com/maenDisease/Solana/files/13171187/bubble.txt)

edit: edited the background color so it would be the original color

**Comment by littlelapin** [comment]
seems like nothing works (even plugins) now thanks to discord pushing out that big update?

**Comment by SpamixOfficial** [comment]
@littlelapin plugins still work for me on BD

**Comment by S0apyy** [comment]
ok ive forked this repository and cleaned up the theme a bit so it doesnt crash discord or have random error popups(happens on vencord).

If this theme breaks again you can contact me on discord here:
s0apyy (if needed u can put and #0 after the name). i am also on the Vencord and BD server, so if needed you can look me up there with the discord search thing.

so this theme will stay up to date until the developer of the theme gets back and is able to update things again :D 

Theme: 
[solana.theme.txt](https://github.com/maenDisease/Solana/files/13182602/solana.theme.txt)

**Comment by j10a1n15** [comment]
> ok ive forked this repository and cleaned up the theme a bit so it doesnt crash discord or have random error popups(happens on vencord).
> 
> If this theme breaks again you can contact me on discord here: s0apyy (if needed u can put and #0 after the name). i am also on the Vencord and BD server, so if needed you can look me up there with the discord search thing.
> 
> so this theme will stay up to date until the developer of the theme gets back and is able to update things again :D
> 
> Theme: [solana.theme.txt](https://github.com/maenDisease/Solana/files/13182602/solana.theme.txt)

Hello!
For some reason literally nothing works in your fork. Could it be bc of the github links? Idk, it started working after I manually added all code from all files into one file, without the links, so yeah

~~Also maybe unrelated but the title bar doenst work for me and is just~~
![image](https://github.com/maenDisease/Solana/assets/45315647/9a830ce4-c33d-43c8-9983-ed0b6a4d17e8)
Edit: nvm just didnt copy the code over

**Comment by S0apyy** [comment]
github links should work

**Comment by j10a1n15** [comment]
Idk, when I use your attached file, its an empty theme. No CSS works, its just the plain discord theme, no modifications
Only works, like I said, when I copy the code from the diff links into one file

**Comment by S0apyy** [comment]
are you renaming the txt file to a theme.css file

because i can only send it in txt format

**Comment by j10a1n15** [comment]
Yes I am, I can also select that .theme.css file, and bd shows it as "Solana has been enabled", but the css does not load at all

**Comment by S0apyy** [comment]
https://github.com/S0apyy/BetterDiscordStuff/blob/main/Themes/Solana/Solana.theme.css

try this maybe?

**Comment by j10a1n15** [comment]
![image](https://github.com/maenDisease/Solana/assets/45315647/961be447-e9a3-4641-9d39-582d3bd58bb4)
Nope, nothing happens

**Comment by S0apyy** [comment]
weird, probably an issue with bd

**Comment by SpamixOfficial** [comment]
@j10a1n15 
Have you tried with Vencord? As @S0apyy said it might just be a BD issue. Vencord and OpenAsar still seems to be functional.
